modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp

```diff
@@ -204,11 +204,8 @@ _bstr_t getMime(CLIPFORMAT cf)
 int create_mime_stuff()
 {
     addPair(GLASS_TEXT_PLAIN, CF_UNICODETEXT);
-    addPair(GLASS_TEXT_PLAIN_LOCALE, CF_TEXT);
```

Comment:
I would not remove them, rather replace CF_TEXT with CF_UNICODETEXT. We still need to support the MIMEs.